drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/custom_client_demographics_loader.rb

```diff
@@ -69,6 +69,7 @@ def perform
           )
         end
 
+        # FIXME: cell phones being imported as float strings, eg "1112223333.0" unable to reproduce locally
```

Comment:
@ttoomey this is so odd! I re-ran it again in production, from the exact same excel file, and I see the numbers being stored correctly. I guess I'll remove this comment and hope for the best- (i.e. make sure to check them every time...).

Reply:
And for reference, https://github.com/roo-rb/roo/issues/53